src/main/java/org/symphonyoss/symphony/messageml/elements/TableHeader.java

```diff
@@ -43,4 +45,15 @@ public void validate() throws InvalidInputException {
     assertNoText();
     assertContentModel(Collections.<Class<? extends Element>>singleton(TableRow.class));
   }
+
+  @Override
+  public Node asMarkdown() throws InvalidInputException {
+    Element parent = getParent();
+    if (TableSelect.class.equals(parent.getClass())) {
```

Comment:
Instead of doing this Parent thing, why don't we actively implement new classes for the thead within a formSelect?

Reply:
because the properties only exist on TableSelect